src/DotNetCore.CAP.RabbitMQ/RabbitMQConsumerClient.cs

```diff
@@ -172,6 +172,25 @@ private void OnConsumerRegistered(object sender, ConsumerEventArgs e)
             };
             OnLog?.Invoke(sender, args);
         }
+        
+        private byte[] ObjectToByteArray(object obj)
```

Comment:
I don’t think this is a good solution, it will cause extra performance overhead for users who don’t use this feature.